packages/integration-tests/test/actions.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { Config, sleep } from '../../../common'
+import { JsonRpcProvider, Web3Provider } from '@ethersproject/providers'
+import { ganache } from '@eth-optimism/ovm-toolchain'
+import assert = require('assert')
+import * as fs from 'fs';
+
+import {
+  Contract, ContractFactory, providers, Wallet,
+} from 'ethers';
+
+let SimpleStorage
+const L1_USER_PRIVATE_KEY = Config.L1UserPrivateKey()
+const L2_USER_PRIVATE_KEY = Config.L2UserPrivateKey()
+const goerliURL = Config.L1NodeUrlWithPort()
+const optimismURL = Config.L2NodeUrlWithPort()
+const l1Provider = new JsonRpcProvider(goerliURL)
+const l2Provider = new JsonRpcProvider(optimismURL)
+const l1Wallet = new Wallet(L1_USER_PRIVATE_KEY, l1Provider)
```

Comment:
These need to pull from the `config` and also you should be able to use the deployer key

Reply:
`config.DeployerPrivateKey`

Reply:
Which vars are referring to? `L1_USER_PRIVATE_KEY` is pulled from `Config` already. Are you suggesting to use `DEPLOYER_PRIVATE_KEY` instead of `L1_USER_PRIVATE_KEY`?

Reply:
Spoke to Karl and Kevin and got it sorted out so please disregard